PrismLauncher/PrismLauncher
# Rename disabled file on mod duplication

fixes https://github.com/PolyMC/PolyMC/issues/550
To test:
- download a mod 
- disable it (it will appear disabled)
- download the same mod again, the same version
- no more file duplication because of name (not in the folder and not in the list) as we delete one of the resources

Drawbacks: replaces the old file
